dev-python/pyilmbase/pyilmbase-2.2.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 cmake-utils
+
+DESCRIPTION="IlmBase Python bindings"
+HOMEPAGE="http://www.openexr.com"
+SRC_URI="http://download.savannah.gnu.org/releases/openexr/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0/23" # based on SONAME
+KEYWORDS="~x86 ~amd64"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=dev-libs/boost-1.62.0-r1:=[python(+),${PYTHON_USEDEP}]
+	>=dev-python/numpy-1.10.4
+	~media-libs/ilmbase-${PV}:="
+
+DEPEND="${RDEPEND}
+	>=virtual/pkgconfig-0-r1"
+
+RESTRICT="test"
+
+PATCHES=( "${FILESDIR}/${P}-add-missing-files.patch" )
```

Comment:
Why are they missing?

Reply:
For some reason, the https://savannah.gnu.org/ sources have missing files in their tarballs. I filed a bug upstream on that. The GitHub sources do not have missing files, but it is one tarball for all four projects, whereas the savannah sources are separated and have been used for all versions in the past for ebuilds. Should I switch to the official GitHub tarball?

Reply:
Could you link this bug in a comment above the patch? This would help others figure out wtf.

Reply:
As for which sources to use, I'm leaving this up to you. Both options are acceptable, and a third one of rolling your own tarball. I guess that's an overkill though ;-).

Reply:
I like the idea of smaller tarballs, but using ones that are flawed are not good as well. I had to have a big patch for the 2.2.0 release as those tarballs had missing files as well. It seems they didn't get their act together for the new release. Thus, I think the GitHub ones might be a better option, even though it's combined. Making our own would be easy, but overkill.